programs/github

```diff
@@ -23,7 +23,13 @@ end
 
 if action == 'clone' then
 	-- parse input
-	repo, dest = select(2, ...)
+	local repo, treeName, dest = select(2, ...)
+
+	if not dest then
+	  dest = treeName
+	  treeName = 'master'
```

Comment:
I'm not sure but this cloned with tabs and I pretty much hate that lol... tab's < spaces IMO... I am going to open do the format fixes in one commit, because I'm going to mass replace all the tabs.